sps30-uart/sps30.h

```diff
@@ -115,6 +115,23 @@ int16_t sps30_stop_measurement(void);
  */
 int16_t sps30_read_measurement(struct sps30_measurement *measurement);
 
+/**
+ * sps30_sleep() - Enter sleep mode with minimum power consumption.
+ *
+ * Note that this command can only be run when the sensor is in idle
```

Comment:
add a comment that the feature is only available on FW2.0

Reply:
Fixed.